package.json

```diff
@@ -80,7 +80,7 @@
         "react-styleguidist": "^11.0.5",
         "react-test-renderer": "^17.0.0",
         "regenerator-runtime": "^0.13.3",
-        "style-loader": "^2.0.0",
+        "style-loader": "^1.0.0",
```

Comment:
I think we should stay on style-loader and maybe only in the jsdocs generation use then style-loader 1. Do we know what it happens is there any default settings changed from 1 to 2. As for me its strange that a webpack plugin should have any effect on a build version of a JS.

Reply:
Unfortunately I don't know what the exact problem with style-loader@v2 is ... I tried installing style-loader@v3, but then the styleguidist build just threw an error

Reply:
Using style-loader@v2 only for styleguidist solves the problem just partially, because then it would only work for jsdocs.sulu.io and not if a developer runs `npm run styleguide:build` locally

Reply:
Would maybe know great an issue on styleguide to find out why this is happening with style loader 2 and not with style loader 1.
I think would npm run styleguide:build is something which is used that often. Maybe we could also move all about styleguidist out of sulu core to the jsdocs repository.

Reply:
See https://github.com/styleguidist/react-styleguidist/issues/1883

Reply:
i just compared the source code of `jsdocs.sulu.io/2.2` with `jsdocs.sulu.io/2.3`. it looks like the ckeditor styling is not resolved correctly - maybe this is related to our performance issue? 🤔 
https://www.diffchecker.com/aKosEcyC